sample/android-app/build.gradle.kts

```diff
@@ -20,9 +20,16 @@ android {
 }
 
 dependencies {
+    configurations {
+        all {
+            exclude("androidx.lifecycle","lifecycle-viewmodel-ktx")
+        }
+    }
```

Comment:
why?

Reply:
Sample has error on build:
`Duplicate class androidx.lifecycle.ViewModelLazy found in modules lifecycle-viewmodel-2.5.0-runtime (androidx.lifecycle:lifecycle-viewmodel:2.5.0) and lifecycle-viewmodel-ktx-2.2.0-runtime (androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0)`

An exception has been added to resolve the conflict two libs. But, now im add lifecycle-viewmodel-ktx lib in sample and concretize version and problem was solution.